image/templates/helm/stackrox-secured-cluster/README.md.htpl

```diff
@@ -3,24 +3,13 @@
 This Helm chart allows you to deploy the necessary services on a StackRox
 secured cluster: StackRox Sensor, StackRox Collector, and StackRox Admission
 Control.
-
-**PLEASE NOTE:** This Helm chart supersedes the `sensor` Helm chart shipped for previous
-versions of the StackRox Kubernetes Security Platform. If you have previously used the
-`sensor` chart, see the [sensor-chart-upgrade.md](sensor-chart-upgrade.md) document in this
-directory for instructions on how to upgrade.
+If you want to install Secured Cluster Services for Red Hat Advanced Cluster Security,
+refer to [Installing the secured-cluster-services Helm chart](https://docs.openshift.com/acs/installing/installing_helm/install-helm-quick.html#installing-secured-cluster-services-quickly_acs-install-helm-quick).
 
 ## Prerequisites
 
 To deploy the secured cluster services for the StackRox Kubernetes Security Platform, you must:
 - Have at least version 3.1 of the Helm tool installed on your machine
-- Have credentials for the `[< required "" .MainRegistry >]` registry or the other image registry
-  you use.
-
-> **IMPORTANT**
->
-> We publish new Helm charts with every new release of the StackRox Kubernetes
-> Security Platform. Make sure to use a version of this chart that matches the
-> StackRox Kubernetes Security Platform version you have installed.
```

Comment:
I think this is important to keep. What it says is that Central version and Secured Cluster chart version should match so that there are less compatibility problems.

Reply:
Done!